### PR TITLE
Feature/add user group page

### LINKED ISSUE #16: Setup user group page
## Description

blocked by #13 

Add actions column in user groups table and for each row, add a button. 
When we clicked to the button, we go to the user-group page (`/user-group/{group_id}`).

Setup user page with white banner:
- breadcrumb to come back to roles page
-  group name

:warning: This page is independant from previous, so in onMounted you have to make a call to get the object by id.

image
image

### PR DESCRIPTION
**Pull request by k-y0u**
Resolves #16

### CODE REVIEW

File: src/services/UserGroupService.js [hunk 1]
```diff
@@ -12,3 +12,14 @@ export async function find() {
     .then(({ data }) => data.results)
     .catch(manageError);
 }
+
+/**
+ * get userGroup by id.
```
Comment (justinehell) [inline src/services/UserGroupService.js:17]: Get

File: changelog.md [hunk 2]
```diff
@@ -24,3 +24,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * Display all information of a library.
 * Setup User Groups page with:
   * Table to display all user groups.
+* Setup User Group page with:
+  * Display all information of a user Group.
```
Comment (justinehell) [inline changelog.md:28]: - user groups.
- User Group
- user Group
--> inconsistent

**Review by justinehell — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 2 above)
Comment (k-y0u) [reply changelog.md]: done

File: src/services/UserGroupService.js (hunk 1 above)
Comment (k-y0u) [reply src/services/UserGroupService.js]: done

**Review by justinehell — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**